src/EFCore.MySql/Migrations/MySqlMigrationsSqlGenerator.cs

```diff
@@ -391,16 +391,10 @@ protected override void Generate(DropSchemaOperation operation, IModel model, Mi
             builder
                 .Append("CREATE SEQUENCE ")
                 .Append(Dependencies.SqlGenerationHelper.DelimitIdentifier(operation.Name, operation.Schema));
-
-            if (operation.ClrType != typeof(long))
+            if (operation.StartValue <= 0)
             {
-                var typeMapping = Dependencies.TypeMappingSource.GetMapping(operation.ClrType);
-
-                builder
-                    .Append(" AS ")
-                    .Append(typeMapping.StoreType);
```

Comment:
Take a look how [Npgsql](https://github.com/npgsql/efcore.pg/blob/795cb6bdf789ae601c2a483178850c930fbc5d4b/src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs#L1312-L1330) implemented the method.
We should probably implement it in a similar way, so we have to maintain as little custom code as possible.

Reply:
Right, that way is cleaner